### ISSUE TITLE
SEM: Equivalence testing

### ISSUE THREAD

**Issue by bwiernik** [easystats/performance#614]
https://doi.org/10.1080/10705511.2015.1065414

**Comment by strengejacke** [comment]
Is this for _performance_ or _parameters_?

**Comment by DominiqueMakowski** [comment]
based on the abstract, *performance* as it seems to be dealing with the quality of fit of the whole models

**Comment by bwiernik** [comment]
Yeah equivalence testing of approximate fit statistics like RMSEA and SRMR
